satpy/tests/reader_tests/test_nucaps.py

```diff
@@ -208,6 +209,7 @@ def test_load_pressure_based(self):
         ])
         r.create_filehandlers(loadables)
         datasets = r.load(['Temperature',
+                           'Effective_Pressure',
```

Comment:
You'll need to update the total number of loaded datasets in the assertEqual below.